src/components/Col.js

```diff
@@ -26,18 +27,18 @@ module.exports = React.createClass({
 	},
 	getInitialState: function() {
 		return {
-			windowWidth: window.innerWidth
+			windowWidth: root.innerWidth
```

Comment:
I think we could just use a simple ternary here, rather than requiring another package:

```
windowWidth: typeof window !== 'undefined' ? window.innerWidth : 0
```